vanilla-todo/main.js

```diff
@@ -0,0 +1,95 @@
+// Import elements
+const toDoForm = document.getElementById("todo-form"),
+  toDoTitle = toDoForm.querySelector("h3"),
+  toDoInput = toDoForm.querySelector("input"),
+  toDoList = document.getElementById("todo-list");
+
+// Constants
+const LS_TAG = "todos";
+
+// State
+let toDos = [];
+let isEditing = false;
+let editingId = null;
```

Comment:
```suggestion
// State
const state = {
  toDos: [],
  isEditing: false;
  editingId: null;
};
```
이런 식으로 state로 묶어서 표현해줘도 좋을 것 같아요!